Northstar.Client/mod/resource/northstar_client_localisation_english.txt

```diff
@@ -297,6 +297,9 @@ Press Yes if you agree to this. This choice can be changed in the mods menu at a
 		"INGAME_PLAYERS" "Players: ^6BA6C400%s1"
 		"TOTAL_SERVERS" "Servers: ^C46C6C00%s1"
 
+		"MISSING_MOD" "Missing mod \"%s1\" v%s2"
+		"WRONG_MOD_VERSION" "Server has mod \"%s1\" v%s2 while we have v%s3"
```

Comment:
why "we", not "you"

Reply:
idk bro i'm just moving things over to localisation instead of hardcoded in script. Both are grammatically correct.

Reply:
I think its a perfect time to change it

Reply:
```suggestion
		"WRONG_MOD_VERSION" "Server has mod \"%s1\" v%s2 while you have v%s3"
```